deploy/cw-bundle.yaml

```diff
@@ -11318,7 +11320,7 @@ spec:
           value: percona-xtradb-cluster-operator
         - name: DISABLE_TELEMETRY
           value: "false"
-        image: percona/percona-xtradb-cluster-operator:1.15.0
+        image: perconalab/percona-xtradb-cluster-operator:main
         imagePullPolicy: Always
```

Comment:
why do we have main here?

Reply:
Should we update initContainer too? 
not critical at all but can be confusing

Reply:
We need to change percona/percona-xtradb-cluster-operator:1.15.0-logcollector-fluentbit3.1.4 and backup images?

Reply:
it was a mistake, fixed main. other images will be updated by @eleo007